### ISSUE TITLE
[Question] Undescriptive debug messages

### ISSUE THREAD

**Issue by papiezak** [Wohlstand/libADLMIDI#182]
Not an issue, rather  a question.
Sometimes demo player produces something like that during playback:
![nani](https://user-images.githubusercontent.com/32672779/46420714-e3435b00-c730-11e8-9852-ab91da3d4966.PNG)
SySex messages are understandable enough, but
`[9] Playing missing instrument 105`
 `[9] Playing missing instrument 110`...
of which GS/XG bank? GM instruments banjo and fiddle are present and not muted, so what do these mean?

**Comment by Wohlstand** [comment]
Looks like the instrument is not exists in the bank that was referred here, and the fallback to GM. Which bank was requested to play those instruments? Can you send the MIDI file for analyze?

**Comment by Wohlstand** [comment]
But okay, I see in the code a check that will print THIS message when playing really blank instrument.

BUT!

Looking on channel 9, it's just a missing drum in the chosen set!

**Comment by papiezak** [comment]
Looking at SC-8820 manual 105 and 110 would be Jazz Snare 1 and Power Snare 2 if drumkit is Standard then... I will check it

**Comment by Wohlstand** [comment]
Yeah, The DMXOPL bank lacks SC-8820'th set which was noted by my friend and I have made an issue for that to note for SC-8820 set
